stable/osg-frontier-squid/osg-frontier-squid/templates/deployment.yaml

```
@@ -50,12 +50,26 @@ spec:
         - name: SQUID_CACHE_MEM
           value: "{{ .Values.SquidConf.CacheMem }}"
         volumeMounts:
+        {{ if .Values.ExtraConf.Enabled }}
+        - name: osg-frontier-squid-{{ .Values.Instance }}-extra-conf
+          mountPath: /etc/osg/image-config.d/99-extra-squid-params.sh
+          subPath: 99-extra-squid-params.sh 
+        {{ end }}
         # Storage location of the cache data
         - mountPath: /var/cache/squid/osg-frontier-squid-{{ .Values.Instance }}-cache
           subPath: osg-frontier-squid-{{ .Values.Instance }}-cache
           name: osg-frontier-squid-{{ .Values.Instance }}-data
           # Shared directory for logs with the Fluent-bit sidecar, can be enabled or disabled from values.yaml
       volumes:
+        {{ if .Values.ExtraConf.Enabled }}
+        - name: osg-frontier-squid-{{ .Values.Instance }}-extra-conf
+          configMap:
+            name: {{ template "osg-frontier-squid.fullname" . }}
+            items:
+            - key: extra-config.sh
+              path: extra-config.sh
+              mode: 256
```

```suggestion
              mode: 448
```

executable file mode